initializr-web/src/main/java/io/spring/initializr/web/project/ProjectRequest.java

```diff
@@ -66,7 +66,7 @@ public void setDependencies(List<String> dependencies) {
 	}
 
 	public String getName() {
-		return this.name;
+		return cleanInputValue(this.name);
```

Comment:
I think it would be more natural to do this in `DefaultProjectRequestToDescriptionConverter`, perhaps in a way that makes it available to sub-classes if necessary.

Reply:
Yes Make sense, To keep original request as it is, and override in changes in DefaultProjectRequestToDescriptionConverter

Reply:
Updated code for same